Associations/rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+  end
+
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Similarly, `book` and `person` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _modify (set)_ it. Kindly ensure to improve upon this aspect.